.phpunit.result.cache

```diff
@@ -0,0 +1 @@
ts\\Routes\\Wiki\\Managers\\CreateTest::testOptionsRequestResponds200":0.014,"Tests\\Routes\\Wiki\\Managers\\DetailsTest::testOptionsRequestResponds200":0.015,"Tests\\Routes\\Wiki\\Managers\\ListTest::testOptionsRequestResponds200":0.012,"Tests\\Routes\\Wiki\\Managers\\MineTest::testOptionsRequestResponds200":0.013,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testSetInvalidSetting":0.076,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingNoWiki":0.013,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #0":0.014,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #1":0.015,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #2":0.014,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #3":0.014,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #4":0.015,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #5":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #6":0.016,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #7":0.014,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #8":0.013,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #9":0.015,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSetting with data set #10":0.011,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #0":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #1":0.015,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #2":0.018,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #3":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #4":0.013,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #5":0.011,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #6":0.011,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #7":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #8":0.011,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #9":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #10":0.012,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #11":0.013,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testValidSettingBadValues with data set #12":0.01,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testOptionsRequestResponds200":0.014,"Tests\\Routes\\Wiki\\Managers\\SettingUpdateTest::testPostRequestWhenUnauthenticatedRespondes401":0.014}}
```

Comment:
We don't want to commit this file (and should probably add it to the .gitignore)